cmd/kubectx/fzf.go

```diff
@@ -23,14 +23,20 @@ type InteractiveSwitchOp struct {
 func (op InteractiveSwitchOp) Run(_, stderr io.Writer) error {
 	// parse kubeconfig just to see if it can be loaded
 	kc := new(kubeconfig.Kubeconfig).WithLoader(kubeconfig.DefaultLoader)
+	defer kc.Close()
```

Comment:
is the rest of the codebase moving defer here? I suspect if we fail doing Parse(), there's not much to close?

Reply:
Thanks, I realize this. Part of my learning is always to do "defer" ...